File: src_simple/f90/somme.f90
! somme  -  11.03.2014

! ******************************************
! Calcul de la somme des n premiers entiers
! ******************************************

program prog_somme

    implicit none

    integer :: i, n, somme

    ! lecture de n
    write(*, *) "n ="
    read(*,*) n
    write(*, *) "n = ", n

    ! initialisation
    somme = 0

    ! boucle
    do i = 1, n, 1
        somme = somme + i
    end do

    ! résultat
    write(*, *) "somme = ", somme

end program prog_somme

File: src_simple/python/somme.py
#!/usr/bin/env python
# -*- coding=utf-8 -*-

""" somme  -  11.03.2014 """

def prog_somme(n):
    """ Calcul de la somme des n premiers entiers """

    # initialisation
    somme = 0.0

    for i in range(n + 1):
         somme += i

    # ou somme = sum(range(n + 1))

    return somme

if __name__ == "__main__":
    # lecture de n
    n = int(raw_input("entrer n : "))
    print("n = {0}".format(n))

    print("Résultat = {0}".format(prog_somme(n)))

